kiwi/system/uri.py

```diff
@@ -51,7 +51,7 @@ class Uri(object):
     def __init__(self, uri, repo_type=None):
         self.runtime_config = RuntimeConfig()
         self.repo_type = repo_type
-        self.uri = uri
+        self.uri = uri if not uri.startswith(os.sep) else 'file:/' + uri
```

Comment:
we have 'dir:/' and 'iso:/' also as a valid local schemas. As far as I remember we introduced 'file:/' schema usage while coding the `derived_from` attribute for containers, where using `dir:/<path>` for tarball was too just too confusing.

Does it means we can now add uris without schema in the description file as a consequence of this change? I am fine with the change if it is clear that using plain paths in the description file is prohibited or without unexpected side effects.

Reply:
>Does it means we can now add uris without schema in the description file as a consequence of this change?

yes it would be the consequence. But you have a point here, the default type for plain paths should be dir:/ not file:/ I'll change that

I could not see a negative side effect, dir:/ and file:/ translates both into _local_path() so that should be safe